### PR TITLE
adding terms

### PR DESCRIPTION
**Pull request by RyanSibley**
This adds all the entries that OIP sent over yesterday with a few minor edits. I want to make the `Exemption` glossary entry a robust one that includes all 9 exemptions. Right now they are all entered individually. I also need to add some links, but I'll do that in the next round of edits.

### CODE REVIEW

**Comment by adborden** [comment]
I believe you can markup like  `<p>`/`<ul>`/`<li>` tags within the definitions to get better formatting.

**Comment by adborden** [comment]
@hbillings I just threw together https://github.com/18F/beta.foia.gov/tree/adb-glossary-markdown, let me know if you want me to merge it into this branch before you do any edits.

**Comment by adborden** [comment]
@AvivaOskow we updated the formatting. I just wanted to run the list treatment by you.
screenshot from 2017-10-27 12-03-40

screenshot from 2017-10-27 11-59-51

**Comment by adborden** [comment]
Includes implementation for #264

**Comment by adborden** [comment]
Includes form title in #256

**Comment by adborden** [comment]
@hbillings this is ready for review

File: js/util/request_form/domify.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/*
+ * domify
+ *
+ * Converts text to react elements. **Only use for known-safe content.**
+ */
+import React from 'react';
+
+function domify(content) {
+  /* eslint-disable react/no-danger */
+  return <div dangerouslySetInnerHTML={{ __html: content }} />;
```
Comment (hbillings) [inline js/util/request_form/domify.jsx]: the name of this cracks me up every time I come across it.

File: js/util/request_form/form_sections.js [hunk 2]
```diff
@@ -1,167 +1,140 @@
...
 
   {
     id: 'processing_fees',
-    title: 'Processing fees',
-    description: 'Under certain circumstances you may need to pay for processing fees.',
+    title: 'Fee waiver',
+    description: domify(`
+      There is no initial <span data-term="fee waiver">fee</span> required to
+      submit a FOIA request, but the FOIA does allow people requesting records
+      to be charged certain types of fees in some instances. If fees will be at
```
Comment (hbillings) [inline js/util/request_form/form_sections.js]: "If fees will be *an* issue"?

File: js/util/request_form/form_sections.js [hunk 3]
```diff
@@ -1,167 +1,140 @@
...
-    title: 'Delivery method',
-    description: 'How would you like to receive your request?',
+    id: 'expedited_processing',
+    title: 'Request expedited processing',
+    description: domify(`
+      You may be entitled to have your request processed on an <span
+      data-term="expedited processing">expedited basis</span> if you demonstrate a
+      <span data-term="compelling need">“compelling need”</span> or satisfy
+      any additional standards that may be included in the regulations of the
+      agency to which you are submitting your request.  A compelling need
```
Comment (hbillings) [inline js/util/request_form/form_sections.js]: Two spaces after the period, I think.

**Comment by RyanSibley** [comment]
@hbillings @adborden I think OIP wants to demo the glossary feature during the CIO meeting today -- and will they can do it with the few entries in there right now, a more robust list might do a bit more in validating our content strategy. Anyway we can merge this before that meeting?

File: js/util/request_form/domify.jsx (hunk 1 above)
Comment (adborden) [reply js/util/request_form/domify.jsx]: Don't laugh. This is serious. Well, maybe it should be `seriouslySetInnerHTML` 😀

**Pull request merged**